source/includes/starting-compass-paste-string.rst

```diff
@@ -49,8 +49,14 @@
 
    .. step:: Connect to your cluster.
 
-      Click :guilabel:`Save` or :guilabel:`Save & Connect` to navigate to the 
-      |compass-short| :ref:`Home Page <compass-home-screen>`. 
+      Click :guilabel:`Save`, :guilabel:`Connect`, or :guilabel:`Save & Connect` 
```

Comment:
[suggestion]

Consider rephrasing as: "To navigate to the |compass-short| :ref:\`Home Page <compass-home-screen>\`, click :guilabel:\`Save\`, :guilabel:\`Connect\`, or :guilabel:\`Save & Connect\`."